include/iarc7_motion/AccelerationPlanner.hpp

```diff
@@ -0,0 +1,214 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Acceleration Planner
+//
+// Implements receiving twist arrays and the acceleration ramps
+//
+////////////////////////////////////////////////////////////////////////////
+
+#ifndef ACCLERATION_PLANNER_H
+#define ACCLERATION_PLANNER_H
+
+#include <ros/ros.h>
+#include <algorithm>
+#include <vector>
+#include "gtest/gtest_prod.h"
+#include "iarc7_msgs/TwistStampedArrayStamped.h"
+#include "geometry_msgs/TwistStamped.h"
+#include "QuadVelocityController.hpp"
```

Comment:
Probably separate package headers, project headers, and system headers and alphabetize within categories